### ISSUE TITLE
Environment column is not included in Azure Table

### ISSUE THREAD

**Issue by andrebriggs** [samiyaakhtar/container-journey#25]
Previously I say a demo that included `environment` as a column that is loaded up. Is it supposed to live the Azure table?
<img width="927" alt="image" src="https://user-images.githubusercontent.com/3119513/63219413-63469980-c126-11e9-8f47-d9567a31cfaa.png">

**Comment by samiyaakhtar** [comment]
Yes, I need to update documentation to include the right script that includes the env column. Will open issue in new repo to update docs.
